TextArea Overview: Change demo description (#1665)
DevExpress/devextreme-demos: JSDemos/Demos/TextArea/Overview/description.md

> The TextArea enables users to enter and edit a multi-line text. The component can have fixed or resizable [height](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#height). The component with the fixed **height** displays a scroll bar if the entered text exceeds the physical space of the text area. If you set the [autoResizeEnabled](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#autoResizeEnabled) property to **true**, the TextArea automatically resizes its height to fit the text.

Reviewer: ```suggestion
The TextArea component enables users to enter and edit multi-line text. This component can have a fixed or resizable [height](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#height). The component with the fixed **height** displays a scroll bar if the entered text exceeds the text area. If you set the [autoResizeEnabled](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#autoResizeEnabled) property to **true**, the TextArea automatically resizes its height to fit the text.
```

> You can also specify the [maxLength](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#maxLength) property to deny users to enter more than allowed number of characters. Note that this property only limits the number of characters for users, you can still set the text of the desired length programmatically.

Reviewer: 1. Can we update the first sentence as follows?

`You can also specify the [maxLength](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#maxLength) property to limit the number of characters a user can enter.`

2. `Note that this property only limits the number of characters for users, you can still set the text of the desired length programmatically.`

I think we can separate this sentence into two sentences. I'd also like to update the second part of the sentence. <br>
The first sentence is fine:

`Note that this property only limits the number of characters for users.`

In the second sentence, I'm not sure what you mean by "you can still set the text of the desired length".<br>
Are you telling the reader that they can set the length in code? For example:

`You can change the maximum length of the text programmatically.`

Or do you mean that if you set the text in code, you can enter text that exceeds the maximum length? For example:

`You can exceed the maximum character length if you set the text programmatically.`

Author: 1. Okay, let's update this sentence.
2. I mean that if you set the maxLength property, for example, to 100 characters, the original text that is stored in the value property can still be over 100 characters (it can be as long as the programmer wants). The users will see the first 100 characters of the text in this situation. I think your second suggestion is more suitable, but I still think it's a bit off.

Reviewer: Thanks for the clarification. What do you think about the following?

`You can also specify the [maxLength](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#maxLength) property to limit the number of characters a user can enter.

Note that this property only limits the number of characters for users. You can enter text that exceeds the maximum character length programmatically, but the number of characters displayed to users will still be limited by this property setting.`

Let me know what you think.

Author: I changed it. Thanks for your suggestion.

Reviewer: Thank you.

The TextArea stores the text in the [value](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#value) property and updates it after the DOM event occurs. To specify which DOM event to use instead of the default *"change"* event, use the [valueChangeEvent](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#valueChangeEvent) property. To handle the value change, use the TextArea's [onValueChanged](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#onValueChanged) function.

> Try the functionality under the Event Handling and API demo section to see how it works in action. Use the "Synchronize text areas" drop-down menu to select which event updates the read-only component's value: *"change"* or *"keyup"*.

Reviewer: ```suggestion
Change the text in the Event Handling and API section below to see how this feature works. Use the "Synchronize text areas" drop-down menu to select which event updates the read-only component's value: *"change"* or *"keyup"*.
```
